public/index.html

```diff
@@ -2106,6 +2106,117 @@ <h1 class="text-8xl">Tabs</h1>
         </label>
       </button>
     </section>
+
+    <div>
+        <h2 class="text-4xl">Image Upload</h2>
+          <label for="input-file" style="width: 704px; height: 56px" class="block text-center border border-dashed border-accent rounded-md pt-4 pb-5 bg-grey-shd7 cursor-pointer my-10 mx-10">
+            <svg class="inline-block" width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M3.16683 3.33333H11.8335V8H13.1668V3.33333C13.1668 2.598 12.5688 2 11.8335 2H3.16683C2.4315 2 1.8335 2.598 1.8335 3.33333V11.3333C1.8335 12.0687 2.4315 12.6667 3.16683 12.6667H8.50016V11.3333H3.16683V3.33333Z" fill="#00A8E8"/>
+              <path d="M5.8335 7.33333L3.8335 10H11.1668L8.50016 6L6.50016 8.66667L5.8335 7.33333Z" fill="#00A8E8"/>
+              <path d="M13.1668 9.33333H11.8335V11.3333H9.8335V12.6667H11.8335V14.6667H13.1668V12.6667H15.1668V11.3333H13.1668V9.33333Z" fill="#00A8E8"/>
+              </svg>              
+            <input type="file" id="input-file" class="opacity-0 absolute" style="z-index: -1">
+            <span class="text-accent" style="font-weight: 600">Upload one or multiple images</span>
+          </label>
+    </div>
+
+    <div>
+      <h2 class="text-4xl">Uploading Images</h2>
+      <div class="mt-4 ml-4">
+          <img src="./img/image.png" alt="">
+        <div style="width: 704px; height: 40px" class="flex items-center px-4 border-grey-shd5 border rounded-bl-md rounded-br-md mt-1 mb-12">
+          <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M0 4C0 1.79086 1.79086 0 4 0H20C22.2091 0 24 1.79086 24 4V20C24 22.2091 22.2091 24 20 24H4C1.79086 24 0 22.2091 0 20V4Z" fill="#F2F9FF"/>
+            <path d="M14.085 14.4023H9.91504L8.97852 17H7.625L11.4258 7.04688H12.5742L16.3818 17H15.0352L14.085 14.4023ZM10.3115 13.3223H13.6953L12 8.66699L10.3115 13.3223Z" fill="#8D9A9E"/>
+            </svg>
+          <input type="text" placeholder="Type caption (optional)" class="inline-block w-full h-5 text-sm font-sans outline-none ml-4 text-grey-shd1">
+        </div>
+      </div>
+      
+  </div>
+
+  <div>
+    <h2 class="text-4xl">Uploading Images</h2>
+    <div class="mt-4 ml-4">
+      <div class="relative">
+        <img src="./img/image.png" alt="" style="opacity: 0.2; filter: blur(2px)">
+        <div class="rounded-t-md rounded-r-md absolute bottom-0 top-0 left-0 right-0 h-full opacity-80" style="width: 708px; background-color: rgba(237, 232, 252, 0.88);">
```

Comment:
Tailwind should be used instead of native css. for width, you can pick any number close to design from tailwind. As these widths are not final. the component will be controlled by its container later

Reply:
For this specific div, i had to use 708px width to be exactly like the image above it because it's a layer on top of the img. since neither tailwind nor the config file has this particular width I had to use the inline styling.

Reply:
- As discussed with mostafa previously, all these atoms widths are determined by their padding and content. the width will be set later by the container. The point is, you don't have to use the exact width as design when implementing the atoms. Just take care of paddings and content. 

- You have used style tag in different lines other the line we discuss here. All the same case